packages/create-angular/src/app/components/views/RiversView.component.ts

```diff
@@ -0,0 +1,363 @@
+import {
+  Component,
+  computed,
+  ContentChild,
+  effect,
+  signal,
+} from '@angular/core';
+import { Color, Deck, MapViewState, WebMercatorViewport } from '@deck.gl/core';
+import { CardComponent } from '../Card.component';
+import { CardCollapsibleComponent } from '../CardCollapsible.component';
+import { FormulaWidgetComponent } from '../widgets/FormulaWidget.component';
+import { LayersComponent } from '../Layers.component';
+import { LegendEntryCategoricalComponent } from '../legends/LegendEntryCategorical.component';
+import { context } from '../../../context';
+import { AccessTokenService } from '../../services/AccessToken.service';
+import { Map } from 'maplibre-gl';
+import { BASEMAP, VectorTileLayer } from '@deck.gl/carto';
+import {
+  createViewportSpatialFilter,
+  vectorTilesetSource,
+} from '@carto/api-client';
+import { debouncedSignal } from '../../../utils';
+import { HistogramWidgetComponent } from '../widgets/HistogramWidget.component';
+
+const CONNECTION_NAME = 'amanzanares-pm-bq';
+const TILESET_NAME =
+  'cartodb-on-gcp-pm-team.amanzanares_opensource_demo.national_water_model_tileset_final_test_4';
```

Comment:
We'll need to switch back to a public dataset before merging this I think, right? It's a  great example, hopefully the rivers dataset can be added to CARTO's public datasets!

Reply:
Yes! We'll change this to point to the public demo_tilesets dataset :)